Update Alps, remove Todi (#233)
C2SM/c2sm.github.io: docs/alps/vclusters.md

> This would allow standard connections like `ssh santis` but also specifying the login node like `ssh santis-ln002` if needed. Replace `cscsusername` with your actual user name.

Reviewer: ```suggestion
This allows standard connections like `ssh santis`, but you can also specify a login node if needed, e.g., `ssh santis-ln002`. Replace `cscsusername` with your actual username.
```

## Santis

> The vCluster `santis` is dedicated to **Climate and Weather**. It might, at the beginning, only host [EXCLAIM :material-open-in-new:](https://c2sm.ethz.ch/research/exclaim.html){:target="_blank"} and related projects.

Reviewer: ```suggestion
The vCluster `Santis` is dedicated to **Climate and Weather** and may initially host only [EXCLAIM :material-open-in-new:](https://c2sm.ethz.ch/research/exclaim.html){:target="_blank"} and related projects.
```

Author: I tried to come up with a unified reference to the vClusters, since we call them

- Säntis
- Santis
- `santis`

I would not use `Santis`, however.

Reviewer: You are right, it's better to unify. I'm happy with any of the versions.

Author: Awesome, I will use `santis` etc. to also keep consistency with CSCS docs.

### Deployment Status

> By the end of 2024, the deployment is complete only to ~95 %. 

Reviewer: ```suggestion
By the end of 2024, the deployment will be approximately 95% complete.
```

Author: Actually I wanted to refer to the current state. So we could also write: "At the moment, ..."

Reviewer: That would make more sense. Maybe something like "As of the current state (end of 2024), the deployment..."?

### Differences to the Environment on `todi`

> - `$HOME` is now on a new NFS file system
>     - Your folder `/users/$USER` will be mostly empty
>     - The NFS system still requires some fine-tuning and file system performance might be low.
>         - We recommend to do (especially heavy tasks) on `$SCRATCH`
> - The `$HOME` of `todi` is mounted in `/users.OLD/$USER`
>     - ⚠️ The mount is read-only!
>     - You are responsible to copy your data from `/users.OLD/$USER` :material-arrow-right-thin: `/users/$USER/...`
>     - The mount is temporary - will be removed towards end of January 2025.

Reviewer: ```suggestion
- `$HOME` is now on a new NFS file system
    - Your folder `/users/$USER` will initially be mostly empty
    - The NFS system still requires fine-tuning, and file system performance may be low.
    - We recommend running tasks, especially heavy ones, on $SCRATCH.
- `Todi`'s $HOME is mounted as `/users.OLD/$USER`.
    - ⚠️ The mount is read-only!
    - You are responsible for copying your data from `/users.OLD/$USER` to `/users/$USER/...`.
    - The mount is temporary and will be removed by the end of January 2025.
```

!!! info

>     Despite the need to work on the deployment in the upcoming days, users are invited to already access the system and start familiarising yourself with it and they might also start the data migration of your old home.

Reviewer: ```suggestion
    Despite the need to work on the deployment in the upcoming days, users are invited to already access the system and start familiarising themselves with it and they might also start the data migration of their old home.
```

### Uenvs

To find and use already existing uenvs from `todi`, you need to modify the `CLUSTER_NAME` environment variable.

```shell
export CLUSTER_NAME=todi
uenv image find
```

| uenv                       | activity                       |
|----------------------------|--------------------------------|
| `icon-wcp/v1:rc4`          | build and run ICON             |
| `netcdf-tools/2024:v1-rc1` | pre- and post-processing tools |

### Storage

!!! note "TODO"

The migration of the previous storage is not yet finished. Once there is an update from CSCS, we will inform you here. Also note that the environment variables `$STORE` and `$PROJECT` are not yet set.

## Daint

Daint (Alps) is the vCluster dedicated to the **User Lab**. It is currently accessible at `daint.alps.cscs.ch` (until the current Piz Daint gets decommissioned), so connect with `ssh daint.alps` with the `ssh` settings above.

Even though Climate and Weather also has the dedicated vCluster `santis` (see [below](#santis)), traditional projects might also land on Daint.

### Uenvs

> Same as on `santis`, one can access the uenvs from `todi`:

Reviewer: ```suggestion
As on `santis`, you can access the uenvs from `todi`:
```
